spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1083,113 +1083,70 @@ being set to `true`.
 
-.Spring Cloud Data Flow Metrics Architecture
-image::{dataflow-asciidoc}/images/dataflow-metrics-arch.png[Spring Cloud Data Flow Metrics Architecture , scaledwidth="100%"]
+There are two versions of the Metrics Collector, a 1.0 version based on Spring Boot 1.0 that understands how to aggregate metrics from Spring Boot 1.x applications and a 2.0 version based on Spring Boot 2.0 that understands how to aggregate metrics from Spring Boot 1.0 and 2.0.
+There is a Metrics Collector server for Rabbit and Kafka.
+You can find more information on downloading and running the Metrics Collector on its https://github.com/spring-cloud/spring-cloud-dataflow-metrics-collector[project page].
 
-As with the App Starters, there is a Spring Boot uber jar artifact of the Metrics Collector for all of the supported binders.
-You can find more information on building and running the Metrics Collector on its https://github.com/spring-cloud/spring-cloud-dataflow-metrics-collector[project page].
 
-The dataflow server now accepts an optional property: `spring.cloud.dataflow.metrics.collector.uri`. This property should point to the URI of your deployed
-metrics collector app. For example, if you run the metrics collector locally on port `8080` then start the server (local example) with the following command:
+The Data Flow server property: `spring.cloud.dataflow.metrics.collector.uri` references the URI the Metrics Collector.
+For example, if you run the Metrics Collector locally on port `8080`, this is how you start local Data Flow server to reference the Metrics Collector.
 
 [source,bash,subs=attributes]
 ----
 $ java -jar spring-cloud-dataflow-server-local-{project-version}.jar --spring.cloud.dataflow.metrics.collector.uri=http://localhost:8080
 ----
 
-The Metrics Collector can be secured with 'basic' authentication that requires a username and password. To set the username and password, use the properties `spring.cloud.dataflow.metrics.collector.username` and `spring.cloud.dataflow.metrics.collector.password`.
+The Metrics Collector can be secured with 'basic' authentication that requires a username and password.
+To set the username and password, use the properties `spring.cloud.dataflow.metrics.collector.username` and `spring.cloud.dataflow.metrics.collector.password` when starting the Data Flow server.
 
-The metrics
-for each application are published when the property `spring.cloud.stream.bindings.applicationMetrics.destination` is set.  This can be set as any other
-application property when deploying an application in Data Flow.  Since
-it is quite common to want all applications in a stream to emit metrics,
-setting it at the Data Flow server level is a good way to achieve that.
+The metrics for each application are published when the property `spring.cloud.stream.bindings.applicationMetrics.destination` is set.
+Using a destination name of `metrics` is a good choice as the Metrics Collector subscribes to that name by default.
```

Comment:
Also, add in the end: ", which of course, can be overridden to be different than the default as needed"